### PR TITLE
[setup.py] add author & author_email

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[metadata]
+name = tool
+version = attr: tool.version.__version__
```
Comment (lachmanfrantisek) [inline setup.cfg]: Just like Tomas sad in other PR, the setuptools_scm would be nice, but we can leave this for CI/testing/tooling discussion.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[metadata]
```
Comment (lachmanfrantisek) [inline setup.cfg]: Nice to this instead of `setup.py`.

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3
+    Topic :: Software Development
+keywords = git, github
+
+[options]
+packages = find:
```
Comment (lachmanfrantisek) [inline setup.cfg]: I do not know this!

Is it possible to also exclude packages (e.g. tests)? (It is only a matter of time to add them.)

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
[the remark this answers is not in the document]
Comment (jpopelka) [reply LICENSE]: I've just added missing license file for `license='GPLv3+'` field in `setup.py` (now `setup.cfg`). Potential license change should be discussed in different issue/PR.

File: setup.cfg (hunk 1 above)
Comment (jpopelka) [reply setup.cfg]: I'd rather see such a change in different PR.

File: setup.py [hunk 5]
```diff
@@ -1,51 +1,10 @@
 #!/usr/bin/python3
 # -*- coding: utf-8 -*-
 
-import os
-
+from pathlib import Path
 from setuptools import find_packages, setup
```
[the remark this answers is not in the document]
Comment (jpopelka) [reply setup.py]: removed

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+#[options.data_files]
+#share/colin/rulesets =
+#  rulesets/default.json
```
[the remark this answers is not in the document]
Comment (jpopelka) [reply setup.cfg]: removed

File: setup.cfg (hunk 3 above)
Comment (jpopelka) [reply setup.cfg]: added commented out section for excluding `tests/`

File: LICENSE (hunk 4 above)
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply LICENSE]: @lachmanfrantisek not a problem

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply LICENSE]: @TomasTomecek thanks, I am a little bit scared of these legal things...

**Pull request merged**